website/docs/docker/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Overview
+description: Docker Concepts and Execution Overview
+sidebar_position: 0
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import ThemedImage from '@theme/ThemedImage';
+
+To make the analysis process scalable, reproducible, and cost-efficient,
+GATK-SV is designed as a cloud-native pipeline, 
+meaning it runs on virtual machines (VMs) hosted in the cloud.
+These VMs are pre-configured with all the necessary tools, scripts, and settings 
+required to run the GATK-SV analysis reliably. 
+
+
+To ensure that the analysis can be easily replicated and shared, 
+GATK-SV utilizes Docker technology. 
+Docker allows the tools and scripts, including all their dependencies and configurations, 
+to be packaged into a self-contained unit called a container. 
+This container can be deployed and run on different VMs in the cloud, 
+making the analysis process consistent and reproducible across multiple experiments or collaborations.
+
+
+Docker containers are built from Docker images, 
+which serve as the blueprints or templates for creating containers.
+Dockerfiles are used to define the contents and behavior of a Docker image. 
+A Dockerfile is a text file that contains a series of instructions, 
+specifying the base image, adding dependencies, configuring settings, 
+and executing commands necessary to build the desired software environment within the container.
+
+
+The following figure is a high-level illustration depicting the relationship 
+between Dockerfiles, Docker images, Docker containers, and Cloud VMs.
+
+
+<ThemedImage
+  alt="Docker Infrastructure Diagram"
+  sources={{
+    light: useBaseUrl('/img/docker_infra_diagram.png'),
+    dark: useBaseUrl('/img/docker_infra_diagram.png'),
+  }}
+/>
+
+
+The GATK-SV Docker setup is organized as follows:
+
+ - **Dockerfile**: There are 21 Dockerfiles available in the GitHub repository. 
```

Comment:
```suggestion
 - **Dockerfile**: 
```
I don't think it's necessary to cite the specific number here. Also it could change and then we'd need to remember to change it here.